hackathon/tests/test_views.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from hackathon.models import Hackathon
+
+
+class TestHackathonViews(TestCase):
+    """Tests views for the Hackathon app."""
+
+    def setUp(self):
+        """Sets up the models for testing"""
+        user = User.objects.create(username="testuser")
+        user.save()
```

Comment:
I think you can remove `user.save()`. The model is saved automatically when you call `create()`. Might have overlooked that in your previous tests. If you have used the same there you can go back in another PR and change that as well.

Reply:
Will do!